c/utils/terminal/terminal.cc

```diff
@@ -163,11 +183,22 @@ void Terminal::forget_window_size() {
 }
 
 void Terminal::_detect_window_size() {
+#if DT_OS_WINDOWS
+  CONSOLE_SCREEN_BUFFER_INFO csbi;
+  HANDLE h = GetStdHandle(STD_OUTPUT_HANDLE);
+  int ret = GetConsoleScreenBufferInfo(h, &csbi);
+  bool is_size_not_detected = (ret == 0);
+  width_ = csbi.srWindow.Right - csbi.srWindow.Left + 1;
+  height_ = csbi.srWindow.Bottom - csbi.srWindow.Top + 1;
+#else
```

Comment:
macros should be indented the same way as if they were regular C++ `if` statements, in order to allow proper code folding.

Reply:
Sure, I've adjust the macros indentation. Don't use the folding much...